src/test/resources/http-tls-1.3-test.xml

```diff
@@ -0,0 +1,76 @@
+    <http:listener-config
+            name="HTTP_Listener_config_unsecure" doc:name="HTTP Listener config">
+        <http:listener-connection host="0.0.0.0"
+                                  port="${internal.port.1}"/>
+    </http:listener-config>
+    <http:request-config
+            name="HTTP_Request_configuration"
+            doc:name="HTTP Request configuration">
+        <http:request-connection protocol="HTTPS"
+                                 host="tls13.akamai.io" port="443">
+            <tls:context enabledProtocols="TLSv1.3"
+                         enabledCipherSuites="TLS_AES_128_GCM_SHA256">
+                <tls:trust-store path="tls13/myTruststore.jks"
+                                 password="123456"/>
+                <tls:key-store type="pkcs12" path="tls13/cert-localhost.p12"
+                               keyPassword="123456" password="123456"/>
+            </tls:context>
+        </http:request-connection>
+    </http:request-config>
+    <http:listener-config
+            name="HTTP_Listener_config_secured" doc:name="HTTP Listener config"
+    >
+        <http:listener-connection protocol="HTTPS"
+                                  host="0.0.0.0" port="${internal.port.2}">
+            <tls:context enabledProtocols="TLSv1.3"
+                         enabledCipherSuites="TLS_AES_128_GCM_SHA256">
+                <tls:trust-store path="tls13/myTruststore.jks"
+                                 password="123456"/>
+                <tls:key-store type="pkcs12" path="tls13/cert-localhost.p12"
+                               keyPassword="123456" password="123456"/>
+            </tls:context>
+        </http:listener-connection>
+    </http:listener-config>
+    <http:request-config
+            name="HTTP_Request_configuration1"
+            doc:name="HTTP Request configuration"
+    >
+        <http:request-connection protocol="HTTPS"
+                                 host="localhost" port="${internal.port.2}">
+            <tls:context enabledProtocols="TLSv1.3"
+                         enabledCipherSuites="TLS_AES_128_GCM_SHA256">
+                <tls:trust-store path="tls13/myTruststore.jks"
+                                 password="123456"/>
+                <tls:key-store type="pkcs12" path="tls13/cert-localhost.p12"
+                               keyPassword="123456" password="123456"/>
+            </tls:context>
+        </http:request-connection>
+    </http:request-config>
+    <flow name="tls-1.3-client-external" doc:id="e82d891f-3f22-4d51-ba87-4b9bdd140c6b">
+        <http:listener doc:name="Listener" config-ref="HTTP_Listener_config_unsecure" path="/client-external"/>
+        <http:request method="GET" doc:name="Akamai" config-ref="HTTP_Request_configuration"
+                      url="https://tls13.akamai.io/"/>
+        <logger level="INFO" doc:name="Logger" message="#[payload]"/>
+    </flow>
```

Comment:
We are not using this flow and its associated configs. I understand we don't want to use it for automated test to avoid depending on connections to external resources. Maybe your intention was to leave it for manual troubleshooting. In that case, shouldn't we at least comment it out and leave an explanation on how and when to enable it for debugging? I'm concerned it could be misleading as to what we are testing if we leave it like this, what do you think?

Reply:
I was just checking if the test would pass with the pipeline's JDK :P Sorry! Now that it passed I will clean everything up